portal/static/portal/sass/partials/_utils.scss

```diff
@@ -204,7 +204,33 @@ iframe {
   margin-right: 30px;
   height: 800px;
 }
+#addStudents {
+  display: block;
```

Comment:
This can't be re-used. Better names and less specific please. Bootstrap provides a class called `d-block` too that does this

Reply:
Don't know if classes can be used in this case though, have a look